components/file_attachment/file_attachment.test.tsx

```diff
@@ -16,115 +16,162 @@ jest.mock('utils/utils.jsx', () => {
     };
 });
 
-function createComponent({fileInfo, handleImageClick, index, compactDisplay, canDownloadFiles = true, enableSVGs = false} = {}) { //eslint-disable-line react/prop-types
-    const fileInfoProp = fileInfo || {
-        id: 1,
+describe('FileAttachment', () => {
+    const baseFileInfo = {
+        id: 'thumbnail_id',
         extension: 'pdf',
         name: 'test.pdf',
         size: 100,
+        width: 100,
+        height: 80,
+        has_preview_image: true,
+        user_id: '',
+        create_at: 0,
+        update_at: 0,
+        delete_at: 0,
+        mime_type: '',
+        clientId: ''
```

Comment:
Our linting rules must be less strict in the test files. I'll leave it up to you if you want to add the trailing commas for the objects in this file and the other test files.